src/util/account.js

```diff
@@ -72,17 +72,19 @@ export function validateSeed(seed, validBip39Seed) {
 	if (!seed || seed.length === 0) {
 		return {
 			accountRecoveryAllowed: false,
+			bip39: false,
 			reason: 'A seed phrase is required.',
 			valid: false
 		};
 	}
 
-	const words = seed.split(' ');
+	const words = validBip39Seed ? seed.trimEnd().split('') : seed.split(' ');
```

Comment:
```suggestion
	const words = validBip39Seed ? seed.trim().split(' ') : seed.split(' ');
```

Reply:
If it is a validBip39Seed, it should not has already be validated by [line 65](https://github.com/paritytech/parity-signer/pull/531/files#diff-d1d34e6b6d649b5a460b1bf20a221d1eR65). So there is no heading whitespaces for seed if it is validBip39Seed.

Reply:
second one is true, I missed the whitespace in `split` function

Reply:
I think `.split(' ')` is the more important change here